tfx/examples/chicago_taxi_pipeline/taxi_pipeline_kubernetes_test.py

```diff
@@ -0,0 +1,45 @@
+# Copyright 2020 Google LLC. All Rights Reserved.
```

Comment:
Because this is not yet a suggested way to run things, can you move this to `tfx/orchestration/experimental/kubernetes/examples`?